app/components/alias-component.js

```diff
@@ -0,0 +1 @@
+export { default } from 'ember-data-model-fragments/components/alias-component';
```

Comment:
I think this export should be part of the test dummy app otherwise its going to be exported from the addon.